src/main/java/com/flightstats/hub/dao/aws/DynamoWebhookDao.java

```diff
@@ -28,6 +28,14 @@ public DynamoWebhookDao(AmazonDynamoDB dbClient, DynamoUtils dynamoUtils) {
     }
 
     private void initialize() {
+        if (HubProperties.isReadOnly()) {
+            if (!dynamoUtils.doesTableExist(getTableName())) {
+                String msg = String.format("Probably fatal error. Dynamo webhook config table doesn't exist for r/o node.  %s", getTableName());
```

Comment:
This triplet of lines (msg, log, throw) happens a lot, a helper method somewhere might trim the code down to one-liners in places like this:
```
if (failingCondition) {
    throw ExceptionUtils.newIllegalArgumentException("message goes here", logger);
}
```
```
public class ExceptionUtils {
public static IllegalArgumentException newIllegalArgumentException(String msg, Logger logger) {
    log.error(msg);
    throw new IllegalArgumentException(msg);
}
```
You could even get clever with lambdas and generics, something like (haven't tried this, but something like it should work)
```
// This caller would invoke: throwLoggedException(msg, IllegalArgumentException::new, logger);
public static T throwLoggedException(String msg, Function<String,T> exceptionSupplier, Logger logger) {
    log.error(msg);
    exceptionSupplier.apply(msg);
}
```